src/plugins/discord/config.test.js

```diff
@@ -42,5 +42,6 @@ describe("plugins/discord/config", () => {
       weights: {},
       defaultWeight: 1,
     });
+    expect(parsed.includeNsfwChannels).toEqual(false);
   });
 });
```

Comment:
Would you add the property to the first test case as "true" to test the true case as well

Reply:
I'll do that

Reply:
and done